gax/src/main/java/com/google/api/gax/retrying/ExponentialRetryAlgorithm.java

```diff
@@ -160,8 +161,12 @@ public boolean shouldRetry(TimedAttemptSettings nextAttemptSettings) {
 
   // Injecting Random is not possible here, as Random does not provide nextLong(long bound) method
   protected long nextRandomLong(long bound) {
-    return bound > 0 && globalSettings.isJittered()
-        ? ThreadLocalRandom.current().nextLong(bound)
+    // When jitter is enabled, choose a random value in (bound - bound * jitter_fraction, bound +
```

Comment:
there shouldn't be any enabling or disabling. Just turn it on. Too many options is an antipattern.